common/app/views/fragments/contentMeta.scala.html

```diff
@@ -28,9 +28,8 @@
     }
 
     @if(!(item.content.isGallery || item.content.isImmersive)) {
-
-        @if(!item.content.hasTonalHeaderByline || item.metadata.designType.nameOrDefault == "analysis") {
-            <div class="meta__contact-wrap">
+    <div class="meta__contact-wrap">
+        @if(!item.content.hasTonalHeaderByline && item.metadata.designType.nameOrDefault != "interview" || item.metadata.designType.nameOrDefault == "analysis") {
```

Comment:
A complicated bit of logic here... if we add some brackets and indentation, it might make the intention clearer

```scala
if (
  (
    !item.content.hasTonalHeaderByline &&
    item.metadata.designType.nameOrDefault != "interview"
  ) || 
  item.metadata.designType.nameOrDefault == "analysis"
)
```